Misc/NEWS.d/next/Library/2024-02-11-07-31-43.gh-issue-82062.eeS6w7.rst

```diff
@@ -1,2 +1,3 @@
-Fix :func:`inspect.signature()` to correctly set extension module for
-built-in instance methods.
+Fix :func:`inspect.signature()` to correctly set extension module for built-in
```

Comment:
It describes what the code does, not what the user gets.

And the user gets that the module globals resolution for default values now works not only for global extension functions, but also for methods of extension classes.